tests/scripts/t3000/run_t3000_model_perf_tests.sh

```diff
@@ -100,7 +100,7 @@ run_t3000_stable_diffusion_35_large_tests() {
 
   echo "LOG_METAL: Running run_t3000_stable_diffusion_35_large_tests"
 
-  env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/experimental/stable_diffusion_35_large/tests/test_performance.py -k "t3k_cfg2_sp2_tp2" ; fail+=$?
+  pytest models/experimental/stable_diffusion_35_large/tests/test_performance.py -k "t3k_cfg2_sp2_tp2" ; fail+=$?
```

Comment:
```suggestion
  pytest models/experimental/stable_diffusion_35_large/tests/test_performance.py -k "t3k_cfg2_sp2_tp2"
```

Also, is it one performance test?

Can we do `-m models_performance_bare_metal` instead to capture this one test?

Reply:
Thanks. I'll move the change to another PR so I don't block the update. Is there a place I can find more info on the test markers?